packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -747,6 +752,335 @@ export class ProjectModel {
         }
     }
 
+    async duplicateContent(projectUuid: string, previewProjectUuid: string) {
+        Logger.debug(
+            `Duplicating content from ${projectUuid} to ${previewProjectUuid}`,
```

Comment:
suggestion: I find `clone` a better verb to describe what the code is doing instead of `duplicate`.